packages/serverless/src/gcpfunction/cloud_events.ts

```diff
@@ -72,14 +72,14 @@ function _wrapCloudEventFunction(
       return (fn as CloudEventFunctionWithCallback)(context, newCallback);
     }
 
-    void Promise.resolve()
+    return Promise.resolve()
       .then(() => (fn as CloudEventFunction)(context))
       .then(
         result => {
-          newCallback(null, result);
+          return newCallback(null, result);
```

Comment:
if we write `result => newCallback(null, result)` prettier should format it nicely